.github/workflows/sbt-node-snyk.yml

```diff
@@ -135,10 +135,12 @@ jobs:
                   ${DEBUG_OPTION} \
                   ${PRUNE_OPTION} \
                   --all-projects \
+                  $([[ ${PYTHON_VERSION:0:2} == 3. ]] && echo "--command=python3") \
```

Comment:
nb this line still works if PYTHON_VERSION is not set, or does not exist

Reply:
Is there any clash if the project is currently running a different version of Python?

Reply:
No, this command only evaluates if the user submits a python version of 3.x.x

Reply:
👍